client/src/main/java/com/paypal/selion/reports/reporter/html/HtmlReporterListener.java

```diff
@@ -300,11 +300,13 @@ private String getContent(ITestResult result) {
                     // Attaching ralogId to each of the page title.
                     if ((logLine.getHref() != null) && (logLine.getHref().length() > 1)) {
                         htmllog = "<a href='../" + logLine.getHref() + "' title='" + logLine.getLocation() + "' >"
-                                + logLine.getMsg() + "</a>";
+                                 + "Page Source </a>";
```

Comment:
The intention to hard code this is , irrespective of a message is present or null or empty it will be displayed below the page source link.

I have tested use cases of asserts failing with null , a message and an empty message.
For null the link text will be null, for empty there wont be a link at all. 
This does not break the report but the page source link will be presented irrespective of the message